### ISSUE TITLE
Accessibility/Identity Providers: User unable to select options from dropdown and options not read using tabs/arrows

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak#17698]
### Describe the bug

The user is not able to select options from the dropdown and options are not read when using tabs/arrows.

This is not an issue for `Add  New Provider` in the User federation.

Tested on MacBook, Chrome browser using Chromevox.

### Version

19.0.0

### Expected behavior

All dropdown options should be accessible and read so that the user knows what they are before selecting.

### How to Reproduce?

- Click on Identity Providers from the left-side menu
-  Add a new identity provider
- Navigate to the `Add provider` dropdown and use Chromvox to listen to the options as you use tab/arrows to navigate between them
- Observe that tab/arrow navigation is not working and the options are not read

**Comment by agagancarczyk** [comment]
> The user is not able to select options from the dropdown and options are not read when using tabs/arrows.

Thanks @edewit I have now raised this issue with the Patternfly team -> https://github.com/patternfly/patternfly/issues/5766

**Comment by agagancarczyk** [comment]
I'm closing this issue because once fixed by the Patternfly team there won't be any extra work required.
